src/components/common/NameRequestInfo.vue

```diff
@@ -182,6 +182,10 @@ export default class NameRequestInfo extends Mixins(DateMixin, EnumMixin, NameRe
 ul {
   list-style-type: none;
 
+  li {
+    display: flex;
+  }
```

Comment:
This and flex CSS below are to wrap the value, ie,

![image](https://user-images.githubusercontent.com/10002889/164573136-8cb9648c-b59f-4d2e-9b33-2eae56a5e8f1.png)

Reply:
I reverted this change per Yui.